blazor/datagrid/how-to/customize-filter-popup-using-css.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: Customize filter popup using CSS in Blazor DataGrid | Syncfusion
+description: Learn here all about Customizing filter popup using CSS in Syncfusion Blazor DataGrid component and more.
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# Customize Filter Popup Using CSS
```

Comment:
why this topic is moved in how to section. Move this topic under excel filter

Change the title as Customize the height and width of filter popup

Reply:
title changed and moved the topic to excel filter